build-artifacts/project-template-gradle/app/src/debug/res/values/colors.xml

```diff
@@ -1,4 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<resources>
-    <color name="nativescript_blue">#3c5afd</color>
```

Comment:
the debug layouts depend on this color identifier, could you consider restoring this file?

Reply:
It still ends up in the final project, I've moved it to `test-app/app/src/debug/res/values`

Reply:
That did not seem to be the case when I tried to build a CLI application using the built package from the latest commit in the branch yesterday.

![image](https://user-images.githubusercontent.com/10464986/34041643-a728924a-e1a1-11e7-9cbb-66bcbe1174e5.png)

Reply:
Thank you, I'll check it again

Reply:
Update: you're right, I'll fix it right away